datafusion/core/src/execution/context.rs

```diff
@@ -2714,6 +2728,46 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn sql_table_insert() -> Result<()> {
```

Comment:
Would it be possible to write this as a sqllogictest https://github.com/apache/arrow-datafusion/tree/main/datafusion/core/tests/sqllogictests (after updating it to use the new codepath)?

Reply:
Addressed 👍🏻